docs/readme-sync/v4.0/deploy-as-a-microservice/010 - optimizely-agent.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Optimizely Agent"
+excerpt: ""
+slug: "optimizely-agent"
+hidden: false
+metadata: 
+  title: "Optimizely Agent microservice - Optimizely Full Stack"
+createdAt: "2020-02-21T20:35:58.387Z"
+updatedAt: "2020-07-14T20:51:52.458Z"
```

Comment:
should these dates be updated with this PR? just curious?

Reply:
yes, will do!